commands/group_test.go

```diff
@@ -0,0 +1,152 @@
+package commands
+
+import (
+	"strings"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mmctl/printer"
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlUnitTestSuite) TestChannelGroupEnableCmdF() {
+	s.Run("Enable group constrains with existing team and channel", func() {
+		teamArg := "team-id"
+		mockTeam := model.Team{Id: teamArg}
+		channelPart := "channel-id"
+		mockChannel := model.Channel{Id: channelPart}
+		channelArg := strings.Join([]string{teamArg, channelPart}, ":")
+		group := &model.Group{Name: "group-name"}
+		mockGroups := []*model.Group{group}
+		groupOpts := &model.GroupSearchOpts{
+			PageOpts: &model.PageOpts{
+				Page:    0,
+				PerPage: 10,
+			},
+		}
+
+		s.client.
+			EXPECT().
+			GetTeam(teamArg, "").
+			Return(&mockTeam, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetChannelByNameIncludeDeleted(channelPart, teamArg, "").
+			Return(&mockChannel, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetGroupsByChannel(channelPart, *groupOpts).
```

Comment:
Should I also test for when functions like this return errors?

Reply:
Yes, it would be great if we had some tests checking the error cases for those calls :)

Reply:
Thank you for your feedback, I'll add some tests for these calls